### ISSUE TITLE
The '+' icon is difficult to identify due to its color matching the folder icons

### ISSUE THREAD

**Issue by ScenarioKG** [arpitkh96/AmazeFileManager#4307]
**Describe the Bug**
The '+' icon blends with the folder color, making it difficult to identify.

**Steps to Reproduce**
1. Open the app with the default color scheme.
2. Locate the '+' icon.

**Expected Behavior**
The '+' icon should have a distinct color that contrasts with the folder icons for better visibility.

**Screenshots**
![mainPage](https://github.com/user-attachments/assets/1d5c8b2a-09bd-4ecc-830f-36ed322a5408)

**Smartphone (please complete the following information):**
 - Device: Google Pixel 5
 - OS: Android 14
 - Rooted: No
 - Version: 3.10

**Comment by EmmanuelMess** [comment]
Adding accessibility as some users might have a more difficult time with the contrast issue.

**Comment by Raghad106** [comment]
Hi @EmmanuelMess, I’d like to work on this issue. Is it available?

**Comment by EmmanuelMess** [comment]
@Raghad106 
> Hi [@EmmanuelMess](https://github.com/EmmanuelMess), I’d like to work on this issue. Is it available?

Yes, but please comment how you would fix it first. I recommend reading [Accessiblity: Color & contrast](https://m3.material.io/foundations/designing/color-contrast) first.

**Comment by Raghad106** [comment]
Okay, I will read it first.
